src/game/gui/textinput.c

```diff
@@ -61,7 +61,7 @@ static void textinput_render(component *c) {
         mode = TEXT_DISABLED;
     }
 
-    text_render(&tb->tconf, mode, c->x, c->y, c->w, c->h, tb->buf);
+    text_render(&tb->tconf, mode, c->x, c->y, c->w, c->h, str_c(tb->text));
```

Comment:
text_render_str(), no need to call str_c().

Reply:
Changed to text_render_str().